### DISCUSSION TITLE
Missing documentation for "Share" in "Configuring Permissions"

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#12338] [Archive]
In page [Configuring Permissions](https://docs.directus.io/configuration/users-roles-permissions/#configuring-permissions) we have explanations for `Create`, `Read`, `Update`, `Delete` but not for `Share`. 
What is `Share` for?

**Comment by rijkvanzanten** [comment]
Controls whether or not a user of the role is allowed to share items from the collection to people outside of Directus (cc @erondpowell)

**Reply by Pachat** [reply]
Thank you for the answer. Do you mean sharing items by mail? by sending an URL? or some other way? to not connected users.

**Reply by erondpowell** [reply]
Great questions @Pachat  `If you go to Content > Some-Collection > Some-Item` you will notice a "share" icon in the sidebar on the right. This generates a URL. Follow the URL and it renders out the Item so that anyone in the world (with the URL) can see it. You can also choose to set a password and a limit to the times the Item can be viewed. You are also able to set a Role the the data-share, so it inherits the permissions of the Role and displays Field values and Relational values as the Role would see it.

<img width="835" alt="Screen Shot 2022-03-30 at 9 23 12 AM" src="https://user-images.githubusercontent.com/62306630/160857797-14a6f229-cfd9-4a44-bdc6-132a50869994.png">

Data-sharing's app-guide doc coming very very soon.
Data-sharing's Configuration doc is farther out in the priorities queue and may take some time to deliver.

**Reply by Pachat** [reply]
@erondpowell Waouh! Such a great feature. Directus has so much power :) Thanks a lot.
